myuw/templates/handlebars/card/accounts/husky.html

```diff
@@ -45,15 +60,61 @@ <h4 class="sr-only">Employee Husky Account</h4>
                 </div>
             </li>
             {{/if}}
+	        <li>
+	         {{#if student_husky_card}}
+            <div class="btn btn-default card-action-btn pull-right" style="margin-top:0px;"><a href="{{student_husky_card.add_funds_url}}" target="_blank" data-linklabel="HFS Account Summary" aria-label="Manage Husky account"> Manage account</a></div>
 
-	</ul>
-	{{#if student_husky_card}}
-            <div class="btn btn-default card-action-btn pull-right"><a href="{{student_husky_card.add_funds_url}}" target="_blank" data-linklabel="HFS Account Summary">Add funds to your accounts</a></div>
-
-        {{else if employee_husky_card}}
-                <div class="btn btn-default card-action-btn pull-right"><a href="{{employee_husky_card.add_funds_url}}" target="_blank">Add funds to your accounts</a></div>
-        {{/if}}
-
+            {{else if employee_husky_card}}
+                <div class="btn btn-default card-action-btn pull-right" style="margin-top:0px;"><a href="{{employee_husky_card.add_funds_url}}" target="_blank" aria-label="Manage Husky account">Manage account</a></div>
+            {{/if}}
+            </li>
+        </ul>
+        <!-- Non-Tacoma Husky card case  -->
+        {{else}}
```

Comment:
For future reference, you can probably refactor this template down so that you don't have to repeat this whole bottom section for the non-tacoma case. 

It looks like the only difference is where the last Manage Account button is placed, so you could put `{{#if is_tacoma}}` conditionals around the part that needs to change at the bottom. Basically if tacoma, add `<li>` around the button, otherwise close the `</ul>` and add the button below.

Reply:
Refactored.